applications/nrf_desktop/board_configuration.rst

```diff
@@ -103,6 +103,46 @@ Sample dongle (``nrf5340dk/nrf5340``)
+        Bluetooth LE is configured to use Nordic Semiconductor's SoftDevice Link Layer and Low Latency Packet Mode (LLPM).
+      * The preconfigured ``debug`` configuration does not use the bootloader due to memory size limits.
+        In the ``debug`` configuration, logs are provided through the UART.
+        For detailed information on working with the nRF54L15 DK, see the :ref:`ug_nrf54l15_gs` documentation.
+      * The preconfigured ``release`` configurations use the MCUboot bootloader built in the direct-xip mode (``MCUBOOT+XIP``) and support firmware updates using the :ref:`nrf_desktop_dfu`.
+        All of the release configurations enable hardware cryptography for the MCUboot bootloader.
+        The application image is verified using a pure ED25519 signature.
+        The public key that MCUboot uses for validating the application image is securely stored in the hardware Key Management Unit (KMU).
+        For more details on nRF54L Series cryptography, see :ref:`ug_nrf54l_cryptography`.
+      * The board supports the ``release`` :ref:`nrf_desktop_bluetooth_guide_fast_pair` configuration that acts as a mouse  (``release_fast_pair`` file suffix).
+
+Sample mouse or keyboard (``nrf54l15dk/nrf54l10/cpuapp``)
+      * The configuration emulates nRF54L10 SoC on the nRF54L15 Development Kit (DK).
+      * The build types allow to build the application as a mouse or a keyboard.
+      * Inputs are simulated based on the hardware button presses.
+      * On the nRF54L10 SoC, you can only use the **GPIO1** port for PWM hardware peripheral output.
+        Because of that, on the DK PCA10156 revision v0.8.1, **LED 0** and **LED 2** cannot be used for PWM output.
+        You can still use these LEDs with the PWM LED driver, but you must set the LED color to ``LED_COLOR(255, 255, 255)`` or ``LED_COLOR(0, 0, 0)``.
+        This ensures the PWM peripheral is not used for the mentioned LEDs.
+      * Only Bluetooth LE transport is enabled.
+        Bluetooth LE is configured to use Nordic Semiconductor's SoftDevice Link Layer and Low Latency Packet Mode (LLPM).
+      * In debug configurations, logs are provided through the UART.
+        For detailed information on working with the nRF54L15 DK, see the :ref:`ug_nrf54l15_gs` documentation.
+      * The configurations use the MCUboot bootloader built in the direct-xip mode (``MCUBOOT+XIP``) and support firmware updates using the :ref:`nrf_desktop_dfu`.
+        All of the configurations enable hardware cryptography for the MCUboot bootloader.
+        The application image is verified using a pure ED25519 signature.
+        The public key that MCUboot uses for validating the application image is securely stored in the hardware Key Management Unit (KMU).
+        For more details on nRF54L Series cryptography, see :ref:`ug_nrf54l_cryptography`.
+      * The board supports the ``debug`` :ref:`nrf_desktop_bluetooth_guide_fast_pair` configuration that acts as a mouse (``fast_pair`` file suffix).
```

Comment:
Not part of this PR, can we mention the debug in line no:156 and 173 as ``debug`` as those should also be in double-backticks.